lightning/src/ln/channelmanager.rs

```diff
@@ -8826,7 +8826,7 @@ where
 		let expiration = StaleExpiration::TimerTicks(1);
 		self.pending_outbound_payments
 			.add_new_awaiting_invoice(
-				payment_id, expiration, retry_strategy, max_total_routing_fee_msat
+				payment_id, expiration, retry_strategy, max_total_routing_fee_msat, Some(invoice_request.clone())
```

Comment:
Let's wrap at 100 characters. No need to do it where we already don't, but here we do already.